docs/docs/how-tos/recursion-limit.ipynb

```diff
@@ -175,7 +240,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Success! The graph raised an error as expected - now let's test setting the recursion limit to 4 and ensure that the graph succeeds in this case:"
+    "Note that this time we terminate after the fourth step."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Additional example\n",
```

Comment:
is this example necessary? perhaps we can simplify? is the idea here to show how supersteps are counted towards recursion limit?